controller/helper/process-survey-instances.js

```diff
@@ -12,17 +12,48 @@ const moment = require('moment');
  * @returns {Object} Complience chart data
  */
 function processSurveyInstances (surveys) {
+    const filterSurveyByState = surveys.filter((survey) => {
+        return survey.state === 'completed';
+    });
+
+    const surveyInstanceLength = 0;
+
+    if (filterSurveyByState !== 'undefined' && filterSurveyByState.length > surveyInstanceLength) {
+        return {
+            // Using UTC as the date gets modified to the local time (GMT in this case) when UTC not used.
+            labels: pickDates(filterSurveyByState),
+            datasets: pickTimeLeft(filterSurveyByState)
+        };
+    }
     return {
-        // Using UTC as the date gets modified to the local time (GMT in this case) when UTC not used.
-        labels: surveys.map((survey) => {
-            return moment(survey.startTime).utc().format('MM/DD/YYYY HH:mm');
-        }),
-        datasets: pickTimeLeft(surveys)
+        labels: '',
+        datasets: ''
     };
 }
 
 /**
  * Takes in a Survey Instances and get the % time left to be shown on complience chart
+ * @param {Object} surveys - list of survey instances
+ * @returns {Object} processed list of datetimes
+ */
+function pickDates (surveys) {
+    const dates = surveys.map((survey) => {
+        return moment(new Date(survey.startTime).toISOString()).utc().format('MM/DD/YYYY HH:mm');
+    });
+
+    const lastDate = moment(new Date(surveys[0]
```

Comment:
@ChristianMurphy  Are you talking about this snippet : "const lastDate = moment(new Date(surveys[0]" ?